src/frontend/src/app/core/layout/secondary-navbar/secondary-navbar.component.ts

```diff
@@ -60,7 +61,11 @@ import { DropdownComponent } from '~core/ui';
             </ng-container>
           </app-dropdown>
         } @else {
-          <button (click)="onLoginButtonClicked()" class="btn btn-icon">
+          <button
+            (click)="onLoginButtonClicked()"
+            class="btn btn-icon"
+            [appTooltip]="tooltipText"
```

Comment:
```suggestion
            [appTooltip]="'Log in'"
```

I'd prefer to keep text in the template unless there's a specific reason to put it in the component (such as complex logic that would be difficult to read in the template).